nss-day-cohort-29/nutshell-divine-madness
# Jd branch

Description
Almost completed the news page functionality

Type of change
Please delete options that are not relevant.

  Bug fix (non-breaking change which fixes an issue)
  New feature (non-breaking change which adds functionality)
  Breaking change (fix or feature that would cause existing functionality to not work as expected)
Testing Instructions for Change Made
Checkout branch jdBranch
run grunt
Page will load and display the stored messages in order by Date/Time
Test sorting by editing time on the entries in the json and refreshing
There is an input at the bottom accompanied by a submit button for creating new messages
Test by entering a new message and you will see the message list refresh with the new message at the end
Each message by the user who is logged in will have an "Edit" button next to it
When clicked, a form to edit the message will show up with the text from the database pre populated along with a submit button for the edit
Change the text and hit submit and you will see the change in the json as well as in the displayed messages
Editing a message does not change the original timestamp, thus preserving the conversational order of the messages
Checklist:
[x ] My code follows the style guidelines of this project
[x ] I have performed a self-review of my own code
[ ] I have commented my code, particularly in hard-to-understand areas
[ ] My changes generate no new warnings
[x ] I have added test instructions that prove my fix is effective or that my feature works
